decidim-accountability/app/views/decidim/accountability/admin/results/_form.html.erb

```diff
@@ -54,11 +54,40 @@
 
     <div class="row column">
       <% if @form.proposals %>
-        <%= form.select :proposal_ids,
-                        @form.proposals,
-                        {},
-                        { multiple: true, class: "chosen-select" }
-        %>
+<!-- dataPicker ------------------------------------------------- -->
+<% picker_options= { id: "decidim_accountability_proposals", class: "picker-multiple", name: 'result[proposal_ids]', multiple: true }
+prompt_params= { url: 'proposals', text: "Add proposal"} %>
+<%= form.data_picker(:proposal_objs, picker_options, prompt_params) {|item| { url: 'proposals', text: "##{item.id}- #{item.title}" }} %>
+<script type="application/javascript">
+$(function() {
+  $(document).on("open.zf.reveal", "#data_picker-modal", function () {
```

Comment:
We should put all this code in a separate file instead of it being embedded in the view. The only reason to have javascript embedded in views is when we want to interpolate `ruby` code and that doesn't seem to be happening here.

Reply:
backend guys do ugly things at frontend :cry: